packages/client/package.json

```diff
@@ -107,7 +107,7 @@
     "*.{js,ts,tsx}": [
       "prettier --write",
       "eslint --fix --max-warnings=0",
-      "react-scripts test --bail --findRelatedTests --watchAll=false"
+      "react-scripts test --bail --findRelatedTests --watchAll=false --env=jest-environment-jsdom-sixteen"
```

Comment:
Added this line because the tests that ran on pre-commit failed, while the normal tests succeeded. See: https://stackoverflow.com/a/61039444